api/src/main/java/org/openmrs/module/sync2/client/reader/ParentFeedWorker.java

```diff
@@ -23,11 +26,25 @@ public void process(Event event) {
 		pullService = Context.getRegisteredComponent("sync2.syncPullService", SyncPullService.class);
 		List tags = event.getCategories();
 
-		pullService.pullAndSaveObjectFromParent(
-				SyncUtils.getValueOfAtomfeedEventTag(tags, AtomfeedTagContent.CATEGORY),
-				SyncUtils.getLinks(event.getContent()),
-				SyncUtils.getValueOfAtomfeedEventTag(tags, AtomfeedTagContent.EVENT_ACTION)
-		);
+		TagService tagService = Context.getRegisteredComponent("atomfeed.tagService", TagService.class);
```

Comment:
I think that "atomfeed.tagService" could be stored in some constant in case of someone would like to get this service.

Reply:
Done